src/components/views/dialogs/AccessTokenDialog.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2017 Vector Creations Ltd
```

Comment:
You should specify your copyright here

Reply:
https://github.com/matrix-org/matrix-react-sdk/pull/5355/commits/a3212c0477ec0fc9166a56a49a9579bae4712d5f thanks :D